packages/gas-tank/worlds.json

```diff
@@ -0,0 +1,9 @@
+{
+  "17069": {
+    "address": "0xba0149de3486935d29b0e50dfcc9e61bd40ae095",
+    "blockNumber": 286789
+  },
+  "31337": {
+    "address": "0xc7b0b46dc0a1131adbdb8762a2773f2769abb527"
+  }
```

Comment:
Should we gitignore this?

Reply:
the `31337` part is unnecessary but I think for the other chains it's actually useful to see where the gas tank is deployed to